packages/databases-collections/src/components/collection-fields/clustered-collection-fields.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { TextInput } from '@mongodb-js/compass-components';
+
+import CollapsibleFieldSet from '../collapsible-field-set/collapsible-field-set';
+import FieldSet from '../field-set/field-set';
+
+const HELP_URL_CLUSTERED = 'https://www.mongodb.com/docs/v5.3/core/clustered-collections/';
```

Comment:
Maybe leave a TODO to update this after 6.0?

Reply:
You mean change v5.3 to current?

Reply:
I think it would be just `manual` instead of `v5.3` :)